src/base/configset.lua

```diff
@@ -61,22 +61,30 @@
 --    blocks with terms fully contained by this list will be considered in
 --    determining the returned value. Terms should be lower case to make
 --    the context filtering case-insensitive.
+-- @param onlylocal
+--    If true, don't fetch values from parent configs.
```

Comment:
Yeah, I noticed that one too... it's from one of the earlier commits, when it was in fact "onlylocal", it got renamed, and the meaning changed, but the comment not updated... I'll have to ask the author about that, and fix this one.